### DISCUSSION TITLE
Please add Bypass All Route

### DISCUSSION THREAD

**Discussion by NeuerUser** [shadowsocks/shadowsocks-android#3037] [Q&A]
Please read contributing guidelines. Thanks.

**Is your feature request related to a problem? Please describe.**
For my use case I need to programmatically (using Tasker) deactivate shadowsocks. Deactivating means turning it into a "Bypass all" proxy. I cannot use the custom ruleset for this as this is already in use with a defined set of rules that I need.

**Describe the solution you'd like**
Add a simple route "Bypass all" to the existing route "All", "Bypass LAN", "Bypass Mainland China", etc.

**Describe alternatives you've considered**
An alternative could be that shadowsocks by default bypasses all proxies if not connected to any server. Another alternative could be to add a second custom ruleset, so that I could utilize that to implement a "bypass all" rule.

**Additional context**
There is not really much I could add. Would be very helpful for me if that could be added. The change is a rather simple change, I guess.

**Comment by madeye** [comment]
Try the empty custom route.

**Reply by NeuerUser** [reply]
Thanks for your reply.

The custom route is already on use. My "active" state is using the custom route, my "inactive" state should be "bypass all".

As there is only one custom route, I cannot use it for bypassing all. If there were two custom route, I could switch between my custom route and an empty one.

See also my description above: " I cannot use the custom ruleset for this as this is already in use with a defined set of rules that I need."
